drud/ddev
# Allow signed releases for macOS and Windows, fixes #1626

## The Problem/Issue/Bug:

We want to sign these things in released versions:
* macOS ddev binary
* Windows ddev.exe binary


## How this PR Solves The Problem:

* Use macOS to do the release build
* Sign Windows binary
* Sign macOS binary
* Update release kicker script to do this.

## Manual Testing Instructions:

* Check macOS binary with `codesign -dv  --verbose 4 </path/to/ddev`
* Check installed windows binary (and downloaded binary) with visiting in file explorer and looking at certs.

## Related Issue Link(s):

## Release/Deployment notes:
